cmd/glasskube/cmd/bootstrap_git.go

```diff
@@ -0,0 +1,148 @@
+		argocdVersion := "v2.11.5+1"
+		if objs, err := clientutils.FetchResources(argocdPath); err != nil {
+			fmt.Fprintf(os.Stderr, "\nAn error occurred fetching argo-cd version (will use %v instead): %v\n",
+				argocdVersion, err)
+		} else {
+			for _, obj := range objs {
+				if v, ok, err :=
+					unstructured.NestedString(obj.Object, "spec", "packageInfo", "version"); !ok || err != nil {
+					fmt.Fprintf(os.Stderr, "\nAn error occurred trying to read the argo-cd version "+
+						"(will use %v instead): %v\n", argocdVersion, err)
+				} else {
+					argocdVersion = v
+					break
+				}
+			}
+		}
+		// install argo-cd package
+		argoCdPkg := client.PackageBuilder("argo-cd").
+			WithRepositoryName("glasskube").
+			WithVersion(argocdVersion).
+			BuildClusterPackage()
+		if _, err := install.NewInstaller(cliutils.PackageClient(cmd.Context())).
+			InstallBlocking(cmd.Context(), argoCdPkg, metav1.CreateOptions{}); err != nil {
+			fmt.Fprintf(os.Stderr, "\nAn error occurred installing argo-cd:\n%v\n", err)
+			cliutils.ExitWithError()
+		}
+
+		fmt.Fprintf(os.Stderr, "\nargo-cd package has been installed!\n\nApplying the glasskube argo-cd application...")
+
+		// apply bootstrap/glasskube-application.yaml into the cluster
+		appPath := fmt.Sprintf("%v/bootstrap/glasskube-application.yaml", basePath)
+		if objs, err := clientutils.FetchResources(appPath); err != nil {
+			fmt.Fprintf(os.Stderr, "\nAn error occurred fetching the bootstrap application:\n%v\n", err)
+			cliutils.ExitWithError()
+		} else if client, err := dynamic.NewForConfig(cfg); err != nil {
+			fmt.Fprintf(os.Stderr, "\nAn error occurred initializing the dynamic client:\n%v\n", err)
+			cliutils.ExitWithError()
+		} else {
+			for _, obj := range objs {
+				if _, err = client.Resource(schema.GroupVersionResource{
+					Group:    "argoproj.io",
+					Version:  "v1alpha1",
+					Resource: "applications",
+				}).Namespace(obj.GetNamespace()).Apply(ctx, obj.GetName(), &obj, metav1.ApplyOptions{
+					FieldManager: "glasskube", // TODO not sure if correct
+				}); err != nil {
+					fmt.Fprintf(os.Stderr, "\nAn error occurred applying the bootstrap application:\n%v\n", err)
+					cliutils.ExitWithError()
+				}
+			}
+		}
+
+		fmt.Fprintf(os.Stderr, "\nglasskube argo-cd application applied successfully!\n\n")
+		fmt.Fprintf(os.Stderr, "You have successfully installed Glasskube and ArgoCD in this cluster.\n")
+		fmt.Fprintf(os.Stderr, "Right now, ArgoCD is starting up and will soon sync with your GitOps repo. ")
+		fmt.Fprintf(os.Stderr, "Please note, that this might take a couple of minutes!\n\n")
+		fmt.Fprintf(os.Stderr, "Run glasskube serve to open the Glasskube UI and either open the ArgoCD UI there, "+
+			"or with the command `glasskube open argo-cd` – but of course you can also use the Argo CLI.\n"+
+			"Follow the ArgoCD docs to get and reset the password to log in: "+
+			"https://argo-cd.readthedocs.io/en/stable/getting_started/#4-login-using-the-cli)")
```

Comment:
you can use a raw string constant for this, I think it would look cleaner. see https://github.com/glasskube/glasskube/blob/62f7e830a4bfdfaadfe184534295e96e4700a92f/pkg/bootstrap/bootstrap.go#L65 for example